transport/http/constants.go

```diff
@@ -20,8 +20,18 @@
 
 package http
 
-// ProcedureHeader is the header used by the HTTP outbound to send and receive
-// the procedure name.
-const ProcedureHeader = "YARPC-Procedure"
+const (
+	// CallerHeader is the HTTP header used to indiate the service doing the calling
+	CallerHeader = "YARPC-Caller"
+
+	// TTLMSHeader is the HTTP header used to indicate the ttl in ms
+	TTLMSHeader = "YARPC-TTLms"
```

Comment:
This can't be the header name we settled on? It's okay for headers to be long. `YARPC-TTL-MILLISECONDS` is totally okay.